packages/utils/eslint.config.js

```diff
@@ -0,0 +1,28 @@
+import tsConfig from '@convidera-team/eslint-config-ts-convidera';
+
+export default [
+  // TypeScript configuration
+  ...tsConfig,
+
+  // Project-specific configuration
+  {
+    files: [
+      '**/*.ts',
+      '**/*.tsx',
+      '**/*.js',
+      '**/*.jsx',
+    ],
```

Comment:
in package json the command to run eslint is `eslint --ext .js,.vue,.ts .` Is it still needed to specify `--ext` flag and having `files` configured?

Reply:
probably also worth including `.mts` and `.mjs` files